youtube_dl/extractor/tvanouvelles.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
+
+class TVANouvellesIE(InfoExtractor):
+    _VALID_URL = r'https?://(www\.|)tvanouvelles\.(ca|com|qc)/.*.?/(?P<id>[^/]+)'
```

Comment:
Don't capture groups you don't use. `(www\.|)` that's incorrect. `.*.?` does not make much sense.